src/main/java/uk/gov/hmcts/reform/iacasedocumentsapi/Application.java

```diff
@@ -4,15 +4,17 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cloud.openfeign.EnableFeignClients;
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages = {"uk.gov.hmcts.reform.ccd.document.am.feign.CaseDocumentClient"})
 @EnableFeignClients(basePackages =
     {
     "uk.gov.hmcts.reform.auth",
     "uk.gov.hmcts.reform.authorisation",
     "uk.gov.hmcts.reform.iacasedocumentsapi",
-    "uk.gov.hmcts.reform.document"
+    "uk.gov.hmcts.reform.document",
```

Comment:
why do you need the reform.document?

Reply:
This has been corrected, Kindly review the code again